src/main/java/games/strategy/engine/data/properties/GameProperties.java

```diff
@@ -81,6 +84,9 @@ public Object get(final String key) {
     if (editableProperties.containsKey(key)) {
       return editableProperties.get(key).getValue();
     }
+    if (playerProperties.containsKey(key)) {
+      return playerProperties.get(key).getValue();
+    }
```

Comment:
Interesting point. For now all properties must be unique across player, editable, and constant so order doesn't matter. I don't know that I currently can see a good case for the same property at multiple levels but ya never know.